README.md

```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
 docker-compose -f "docker-compose.yml" up -d --build
 ```
 
-ps: If you have this kind of error “no matching manifest for windows/amd64 in the manifest list entries”, switch to Linux container in Docker Desktop.
+P.S. If you have the kind of error "no matching manifest for windows/amd64 in the manifest list entries", switch to **Linux container** in Docker Desktop.
+
 ## Usage
 
-### Orthanc server
+### Orthanc Server
 
-Once Orthanc is running, use Mozilla Firefox at URL [http://localhost:8042/](http://localhost:8042/) to interact with Orthanc. The default username is `barbecuehealth` and its password is `barbecuehealth`.
+Once Orthanc is running, use Mozilla Firefox at URL <http://localhost:8042/> to interact with Orthanc. The default username is `barbecuehealth` and its password is `barbecuehealth`.
 
-If you want to change login/password, edit file `config/orthanc.json` and add your new login into RegisteredUsers
+If you want to change the login/password, edit the file `config/orthanc.json` and add your new login into `RegisteredUsers`.
```

Comment:
To change the login/password, edit file `config/orthanc.json` and add a new login into `RegisteredUsers`.